lib/trento/application/usecases/sap_systems/dto/health_summary_dto.ex

```diff
@@ -15,6 +15,8 @@ defmodule Trento.Application.UseCases.SapSystems.HealthSummaryDto do
     field :sapsystem_health, Ecto.Enum, values: Health.values()
     field :database_health, Ecto.Enum, values: Health.values()
     field :clusters_health, Ecto.Enum, values: Health.values()
+    field :hana_cluster_id, :string
```

Comment:
They should be uuid

Reply:
And I think we should maybe name it `cluster_id`. I wouldn't start yet putting hana specific details here. Maybe in the future we will have HANA and NW clusters independently, but yet, our code is using the `cluster` name, and not `hana_cluster`

Reply:
done!